### PR TITLE
feat: add dark theme context

### LINKED ISSUE #402: feat: create a context to switch theme
We need a dark theme in our website, for that we need to create a context to get the current state of the theme.

The theme must be auto set according to the time. i.e after 08:00 PM default value should be dark.

Create a sample darkTheme file to test the context.

### PR DESCRIPTION
**Pull request by Shurtu-gal**
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start _your branch_ off _default branch_.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Linked Issues

Fixes #402 

### Description
 - Current Changing time :- 2000hrs 

❤️Thank you!

### Post merge checklist

- [x] Follow steps from the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.
- [x] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: client/src/context/ThemeContext.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React, {
+  useContext,
+  useEffect,
+  useState,
+} from 'react';
```
Comment (120EE0692) [inline client/src/context/ThemeContext.jsx:5]: @Shurtu-gal check the code is prettier checked pass or not?

File: client/src/pages/_app.jsx [hunk 2]
```diff
@@ -1,28 +1,29 @@
...
-import Head from 'next/head';
-import Router from 'next/router';
-
 // Styles
 import '../../public/index.css';
 
+import React, {
+  useEffect,
+  useState,
+} from 'react';
```
Comment (120EE0692) [inline client/src/pages/_app.jsx:7]: @Shurtu-gal check here as well.

**Review by 120EE0692 — CHANGES_REQUESTED** [review]
@Shurtu-gal for now add a logo in the navbar to switch the theme.

image for reference
image

**Review by 120EE0692 — CHANGES_REQUESTED** [review]
@Shurtu-gal add this icon on left side, and make sure to keep left aligned with the black logo.
image

**Comment by Shurtu-gal** [comment]
> @Shurtu-gal add this icon on left side, and make sure to keep left aligned with the black logo. image

Done

**Review by 120EE0692 — APPROVED** [review]

**Comment by 120EE0692** [comment]
@rutajdash merge this code in a new side branch.

**Review by rutajdash — COMMENTED** [review]
Please attach multiple screenshots or a comprehensive video showing the dark mode view on all pages and the toggle animation.

**Comment by 120EE0692** [comment]
> Please attach multiple screenshots or a comprehensive video showing the dark mode view on all pages and the toggle animation.

We have just created a context to maintain and switch the theme.

For switching a button is placed in the navbar. 

Currently the plan is to merge the pull request in a side branch and develop dark theme page by page there.

**Comment by Shurtu-gal** [comment]
> Please attach multiple screenshots or a comprehensive video showing the dark mode view on all pages and the toggle animation.
[Screencast from 23-01-23 06:41:54 PM IST.webm](https://user-images.githubusercontent.com/100484401/214048693-dca96618-3db7-4b6c-ae12-fe96d0e2e6bd.webm)

Currently, only context and button have been made.

**Comment by rutajdash** [comment]
> > Please attach multiple screenshots or a comprehensive video showing the dark mode view on all pages and the toggle animation.
> 
> We have just created a context to maintain and switch the theme.
> 
> For switching a button is placed in the navbar. 
> 
> Currently the plan is to merge the pull request in a side branch and develop dark theme page by page there.

Very well. Please create branch and switch this PRs merge branch.

**Review by rutajdash — APPROVED** [review]

**Pull request merged**